### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by acanbym**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# jest-assertions-workshop
\ No newline at end of file
+# jest-assertions-workshop
```
Comment (missKatiaPunter) [inline README.md:1]: Can we have a meaningful name for the workshop?

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
...
+
+Instructions for this workshop are as follows:
```
Comment (missKatiaPunter) [inline README.md:3]: That is a great introduction, thank you, Anna 😄 !

File: README.md [hunk 3]
```diff
@@ -1 +1,8 @@
-# jest-assertions-workshop
\ No newline at end of file
+# jest-assertions-workshop
+
+Instructions for this workshop are as follows: 
+
+- blah, blah, blah;
```
Comment (missKatiaPunter) [inline README.md:5]: Good use of Markdown list but can we have the actual steps of the workwhop.

**Review by missKatiaPunter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (acanbym) [reply README.md:1]: Sure! 😄

**Review by missKatiaPunter — APPROVED** [review]

**Pull request merged**